<launch>
  <node name="frontier_exploration" pkg="crowdbot_active_slam"
        type="frontier_exploration" output="screen">
    <param name="frontier_size" value="35"/>
  </node>
</launch>
